### PR TITLE
Operators / it-academy.by / JC1-110-22 / Maksim Bondar / Tasks 1 - 6 complete

### PR DESCRIPTION
**Pull request by shiv-qa-auto**
## Pull request state

Actual status for PR:
- [x] implemented
- [x] reviewed
- [x] fixed
- [ ] approved

### CODE REVIEW

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java [hunk 1]
```diff
@@ -37,6 +37,6 @@ public static void main(String[] args) {
     static float getTotalDepositAmount(int depositAmount, int depositYears, int depositAnnualPercentage) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+        return depositAmount+depositAmount * (((float) depositYears * depositAnnualPercentage) / 100);
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java:40]: "Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java [hunk 2]
```diff
@@ -36,6 +36,6 @@ public static void main(String[] args) {
     static int calculateDeviationStandardWeight(int height, int weight) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        return weight - (height - 110);
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java:39]: "Code smell: Magic Number
Solution: Named Constants"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java [hunk 3]
```diff
@@ -35,6 +35,6 @@ public static void main(String[] args) {
     static float getDistance(float milliseconds) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+        return SOUND_SPEED * milliseconds / 3600;
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java:38]: "Code smell: Magic Number
Solution: Named Constants"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java [hunk 4]
```diff
@@ -30,6 +30,10 @@ public static void main(String[] args) {
     static String getPlayingTime(int playingTimeInSeconds) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        int daysCounter = playingTimeInSeconds / 86400;
+        int hoursCounter = (playingTimeInSeconds - daysCounter * 86400) / 3600;
+        int minutesCounter = (playingTimeInSeconds - daysCounter * 86400 - hoursCounter * 3600) / 60;
+        int secondsCounter = playingTimeInSeconds % 60;
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java:36]: "Code smell: Magic Number
Solution: Named Constants"

**Review by rakovets — CHANGES_REQUESTED** [review]
Need to fix

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java (hunk 1 above)
Comment (shiv-qa-auto) [reply src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java]: Fixed, please review

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java (hunk 2 above)
Comment (shiv-qa-auto) [reply src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java]: Fixed, please review

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java (hunk 4 above)
Comment (shiv-qa-auto) [reply src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java]: Fixed, please review

**Review by rakovets — APPROVED** [review]
Approved

**Pull request closed without merging**